### PR TITLE
Cleanup buildpack output

### PR DESCRIPTION
**Pull request by codingjoe**

### CODE REVIEW

**Comment by codingjoe** [comment]
@amureki you can check the output in the travis built

File: post_compile [hunk 1]
```diff
@@ -1,29 +1,45 @@
...
+  sed "s/^/       /"
+}
+
+puts-step() {
+  echo "-----> $*"
+}
+
+puts-warn() {
+  echo " !     $*"
+}
```
Comment (amureki) [inline post_compile:15]: Why we can't just source those from `bin/utils.sh` as well?

**Review by amureki — COMMENTED** [review]
@codingjoe I have a small clarification question there...

File: post_compile (hunk 1 above)
Comment (codingjoe) [reply post_compile:15]: Because this will be executed inside the repo when the build packs is no longer available.

**Review by amureki — APPROVED** [review]

File: post_compile (hunk 1 above)
Comment (amureki) [reply post_compile:15]: @codingjoe right, I've got it. Thank you :)

**Pull request merged**